stable/datadog/templates/system-probe-configmap.yaml

```
@@ -20,6 +20,9 @@ data:
       bpf_debug: {{ $.Values.datadog.systemProbe.bpfDebug }}
       enable_tcp_queue_length: {{ $.Values.datadog.systemProbe.enableTCPQueueLength }}
       enable_oom_kill: {{ $.Values.datadog.systemProbe.enableOOMKill }}
+      {{- if $.Values.datadog.systemProbe.collectDNSStats }}
+      collect_dns_stats: {{ $.Values.datadog.systemProbe.collectDNSStats }}
+      {{- end }}
```

since we set the default value in `values.yaml`, I think we can remove the `{{if}} {{end}}` check

```suggestion
      collect_dns_stats: {{ $.Values.datadog.systemProbe.collectDNSStats }}
```

---

I briefly mentioned this in the commit message. If/when the agent defaults this option to true, the chart won't accidentally override it back to false if we keep the `if` block.   

I'm not sure how we usually handle options like this, so I'm happy to just require a chart upgrade when the default value is flipped if you think this kind of logic is too complicated to go into the chart.

---

but how people cant deactivate it if the default value is "true"?

---

I feel that the simple solution is to update the chart when the default value is updated.

---

That's a good point, I committed your suggestion.